slackEvents.php

```diff
@@ -215,5 +215,21 @@ function register($url, $userid, $in, $request) {
         $this->log->debug("register mail $profile->email $profile->first_name $profile->last_name");
         $r = $this->agenda->updateAttendee($url, $profile->email, $in, $profile->first_name . ' ' . $profile->last_name);
         $this->app_home_page($userid, $request);
-    }    
+
+        $vevent = $this->agenda->getEvent($url)->VEVENT;
+        $datetime = $vevent->DTSTART->getDateTime();
+        $datetime->setTime(19, 0, 0, 0);
```

Comment:
as we discussed, don't set 19h to avoid conflicts

Reply:
Fixed in d8aad75